### PR TITLE
Merging master to feature/plugin-api.

### PR DESCRIPTION
**Pull request by tswhison**

### CODE REVIEW

File: libopae/CMakeLists.txt [hunk 1]
```diff
@@ -43,6 +43,8 @@ target_include_directories(opae-c PUBLIC
 
 include_directories(${libjson-c_INCLUDE_DIRS})
 
+include_directories(${libjson-c_INCLUDE_DIRS})
```
Comment (lzhan55) [inline libopae/CMakeLists.txt]: Is this line a duplicate of the above line?

Comment (tswhison) [reply libopae/CMakeLists.txt]: I think so. Thanks, @lzhan55

**Comment by lzhan55** [comment]
Did you build the OPAE with "cmake -DBUILD_ASE_SAMPLES=on -DCMAKE_BUILD_TYPE=Coverage -DBUILD_ASE_TESTS=on" to make sure ASE gtest built correctly?

**Comment by lzhan55** [comment]
capture_10_02

Who added intel-fpga-bbb samples?

**Comment by tswhison** [comment]
@lzhan55 thanks to Abelardo, the mpf bbb is compiling correctly now.  Could you please check again in your environment?  Thanks

**Comment by r-rojo** [comment]
I think building ASE is outside the scope of this pull request. There are several things that need to be done for ASE

**Comment by lzhan55** [comment]
Agree. We can resolve ASE gtest issues on feature/plugin-api branch.

**Pull request merged**